bin/endorS.py

```diff
@@ -23,7 +23,7 @@
    '''))
 parser.add_argument('samtoolsfiles', metavar='<samplefile>.stats', type=str, nargs='+',
                     help='output of samtools flagstat in a txt file (at least one required). If two files are supplied, the mapped reads of the second file is divided by the total reads in the first, since it assumes that the <samplefile.stats> are related to the same sample. Useful after BAM filtering')
-parser.add_argument('-v','--version', action='version', version='%(prog)s 0.2')
+parser.add_argument('-v','--version', action='version', version='%(prog)s 0.4')
```

Comment:
We should make sure all scripts have some sort of (open) license associated with it and embedded in the script itself.

@maxibor, @aidaanva and @TCLamnidis could you add to your scripts respectively?